spec/rule_spec.rb

```diff
@@ -1,6 +1,35 @@
 require 'spec_helper'
 
 describe RRule::Rule do
+  describe '#next' do
```

Comment:
Do you need to do anything to implement `lazy` or is that free if you implement `Enumerable`?

Reply:
It's free!